RestartManager.js

```diff
@@ -3,41 +3,54 @@ const NativeCodePush = require("react-native").NativeModules.CodePush;
 
 const RestartManager = (() => {
     let _allowed = true;
-    let _restartPending = false;
+    let _restartInProgress = false;
+    let _restartQueue = [];
```

Comment:
Okay, so this makes sense. Basically, we process these requests one by one, with the idea that we may eventually find one where a restart actually happens. If a restart happens at any point, the context gets destroyed and the queue gets wiped (so the restart would only happen once).

Would it make sense to rename these two variables to `_processingRestartRequest` and `_restartRequests`, to make it clear that they're not necessarily restarts?